main/dhcp/dhcrelay.initd

```diff
@@ -20,6 +20,8 @@ start() {
 		IFACES="${IFACES} -i ${i}"
 	done
 
+	checkpath -d -o dhcp:dhcp /var/run/dhcp
```

Comment:
I prefer long options in scripts as they are more readable

Reply:
@ncopa I've updated the PR to contain long options.